docs/en/developers/01_First_steps.md

```diff
@@ -1,6 +1,47 @@
-# Environment configuration
+# Environment configuration (Docker)
 
-**TODO**
+FreshRSS is built with PHP and use a homemade framework, Minz. The dependencies are directly included in the source code and so you don't need Composer.
+
+There are various ways to configure your development environment, the easiest and most supported one is based on Docker. This is the solution documented below. If you already have a working PHP environment, you probably don't need it.
+
+We assume here that you use a GNU/Linux distribution, capable to run Docker. Otherwise, you'll have to adapt the commands to your case.
```

Comment:
```suggestion
We assume here that you use a GNU/Linux distribution, capable of running Docker. Otherwise, you'll have to adapt the commands accordingly.
```